mobile/lib/screens/kya/kya_lessons_page.dart

```diff
@@ -1,44 +1,57 @@
-import 'package:app/models/kya.dart';
+import 'package:app/screens/kya/kya_final_page.dart';
+import 'package:app/themes/app_theme.dart';
 import 'package:auto_size_text/auto_size_text.dart';
-import 'package:cached_network_image/cached_network_image.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_cache_manager/flutter_cache_manager.dart';
 import 'package:flutter_svg/svg.dart';
-import 'package:scrollable_positioned_list/scrollable_positioned_list.dart';
-
+import '../../models/enum_constants.dart';
+import '../../models/kya.dart';
 import '../../services/native_api.dart';
-import '../../themes/app_theme.dart';
 import '../../themes/colors.dart';
 import '../../widgets/custom_shimmer.dart';
-import 'kya_final_page.dart';
 import 'kya_widgets.dart';
 
 class KyaLessonsPage extends StatefulWidget {
-  const KyaLessonsPage(
-    this.kya, {
-    super.key,
-  });
+  const KyaLessonsPage({
+    Key? key,
+    required this.kya,
+  }) : super(key: key);
+
   final Kya kya;
 
   @override
   State<KyaLessonsPage> createState() => _KyaLessonsPageState();
 }
 
-class _KyaLessonsPageState extends State<KyaLessonsPage> {
-  final ItemScrollController itemScrollController = ItemScrollController();
-  final ItemPositionsListener itemPositionsListener =
-      ItemPositionsListener.create();
-
-  double _tipsProgress = 0.1;
-  int currentIndex = 0;
+class _KyaLessonsPageState extends State<KyaLessonsPage>
+    with SingleTickerProviderStateMixin {
+  List<KyaLesson> _kyaLessons1 = []; //1st array to hold all stack
```

Comment:
Could we use descriptive variable names so that we can be able to minimise comments in our code? Thanks!